docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/utils/index.md

```diff
@@ -207,16 +211,33 @@ Next, we have `snowplow__session_identifiers` and `snowplow__user_identifiers`,
 Currently, we only support session and user identifiers found in atomic fields for Redshift/Postgres. We don't support nested level fields for any warehouses, and for BigQuery you will currently need to do the version management yourself. We will be getting around to supporting this extra functionality soon.
 :::
 
-By default, `snowplow__session_identifiers` is set to `atomic: domain_sessionid`, and `snowplow__user_identifiers` is set to `atomic: domain_userid`. This means that the identifiers for sessions and users are expected to be found in the `domain_sessionid` and `domain_userid` fields, respectively.
+By default, `snowplow__session_identifiers` is set to `[{"table": "atomic", "field": "domain_sessionid"}]`, and `snowplow__user_identifiers` is set to `[{"table": "atomic", "field": "domain_userid"}]`. This means that the identifiers for sessions and users are expected to be found in the `domain_sessionid` and `domain_userid` fields, respectively.
 
 If you have more than one session or user identifier, you can specify multiple entries in the map. The order in which you list them determines the precedence that the macro will use to look for these field values, and `COALESCE` them into the common session/user_identifier field. E.g. if you have the following definition for your `user_identifier`:
 
-```json
-{
-  "my_custom_context": "internal_user_id",
-  "atomic": "domain_userid"
-}
-```
+<Tabs groupId="warehouse" queryString>
+<TabItem value="default" label="BigQuery, Databricks, & Snowflake" default>
+<pre><code className="language-json">
+{`
+[
+  {"schema": "my_custom_context", "field": "internal_user_id"},
+  {"table": "atomic", "field": "domain_userid"}
+]
+`}
+</code></pre>
+</TabItem>
+<TabItem value="r&s" label="Redshift & Postgres" default>
+<pre><code className="language-json">
```

Comment:
Please use `redshift+postgres` as the value here so we are consistent with other pages